oasst-shared/oasst_shared/schemas/inference.py

```diff
@@ -256,6 +256,19 @@ class GeneralErrorResponse(WorkerResponseBase):
     error: str
 
 
+class SafetyParameters(pydantic.BaseModel):
+    ...
```

Comment:
what does `...` mean in this context, is this the same as `pass`?

Reply:
Yes it's just a placeholder, will be removed in a minute as I'll add a parameter to enable/disable safety